### PR TITLE
Trackupstream pike

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by bmwiedemann** [comment]
2nd commit message says 'Ocata', but adds "Pike"

File: jenkins/ci.opensuse.org/openstack-trackupstream.yaml [hunk 1]
```diff
@@ -123,6 +124,9 @@
             OBS) OSCAPI="https://api.opensuse.org"
                 OSC_BUILD_ARCH=x86_64
                 case $OBS_PROJECT in
+                    Cloud:OpenStack:Pike)
```
Comment (cmurphy) [inline jenkins/ci.opensuse.org/openstack-trackupstream.yaml]: Should this be Pike* to include the staging project, the way the other cases are on line 130:

**Comment by toabctl** [comment]
> 2nd commit message says 'Ocata', but adds "Pike"

Done

File: jenkins/ci.opensuse.org/openstack-trackupstream.yaml (hunk 1 above)
Comment (toabctl) [reply jenkins/ci.opensuse.org/openstack-trackupstream.yaml]: Done

**Pull request merged**